src/Domain.LinnApps/Requisitions/RequisitionReportService.cs

```diff
@@ -56,6 +60,51 @@ public async Task<ResultsModel> GetRequisitionCostReport(int reqNumber)
             return model;
         }
 
+        public async Task<string> GetRequisitionCostReportAsHtml(int reqNumber)
+        {
+            var requisition = await this.requisitionRepository.FindByIdAsync(reqNumber);
+
+            var totalCost = requisition.Lines.Sum(l => this.WorkOutCost(l.StoresBudgets, l.Qty, l.Part));
+            var totalNetCost = requisition.Lines.Sum(
+                l => this.WorkOutNetCost(l.StoresBudgets, l.Qty, l.Part, l.TransactionDefinition));
+
+            var reportLines = requisition.Lines
+                .Select(l => new RequisitionCostLine
+                                 {
+                                     ReqNumber = l.ReqNumber,
+                                     Cost = this.WorkOutCost(l.StoresBudgets, l.Qty, l.Part),
+                                     Description = l.Part.Description,
+                                     PartNumber = l.Part.PartNumber,
+                                     Qty = l.Qty,
+                                     NetCost = this.WorkOutNetCost(l.StoresBudgets, l.Qty, l.Part, l.TransactionDefinition),
+                                     UnitPrice = l.StoresBudgets?.FirstOrDefault()?.PartPrice
+                                                 ?? l.Part.BaseUnitPrice.GetValueOrDefault()
+                                 })
+                .ToList();
+
+            reportLines.Add(new RequisitionCostLine
+                                {
+                                    ReqNumber = null,
+                                    Description = "TOTAL",
+                                    Cost = totalCost,
+                                    NetCost = totalNetCost,
+                                    Qty = null,
+                                    PartNumber = null,
+                                    UnitPrice = null
+                                });
```

Comment:
I dont think we need  this extra fake line just to include the totals? yoyu are including the totals as their own fields on the header (RequisitionCostReport.cs) anyways on lines 99-100